Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2722,6 +2727,148 @@ extension BrowserViewController: ContextMenuHelperDelegate {
     }
 }
 
+/**
+ A third party search engine Browser extension
+**/
+extension BrowserViewController {
+
+    /**
+    This allows us to find the part of the WKWebview that has access to
+    the inputAcessoryView
+    **/
+    func findWebViewFirstResponder(view: UIView) -> UIView? {
```

Comment:
Since this is a fairly generic method, can this be moved to a `UIViewExtensions.swift` utils class, removing `WebView` from the name?